base/applications/taskmgr/dbgchnl.c

```diff
@@ -207,20 +207,32 @@ typedef int (*EnumChannelCB)(HANDLE, void*, WCHAR*, void*);
  */
 static int enum_channel(HANDLE hProcess, EnumChannelCB ce, void* user, unsigned unique)
```

Comment:
This function (and related code) is Wine-specific (i.e. dead code for us) and synced to a 15 year old version of Wine (i.e. won't in fact work, even if run in Wine today).
We should do one of four things:
* Leave it as is and don't spend time on it
* Entirely remove the feature
* Sync it to current Wine, such that it actually works if run in Wine
* Find a way to make it work on ROS (we have debug channels, but not libwine.so, so it's going to take some rethinking... or it may not be possible)